virtcontainers/container.go

```diff
@@ -791,6 +797,10 @@ func (c *Container) delete() error {
 		return err
 	}
 
+	if err := c.deleteCgroups(); err != nil {
```

Comment:
For a sandbox container, the hypervisor process has been set in its cgroup, before deleteCgroups, you should remove the tasks from it.